src/Tools/dotnet-monitor/LoggingEventIds.cs

```diff
@@ -66,10 +65,7 @@ internal enum LoggingEventIds
         LoadingProfiler = 53,
         SetEnvironmentVariable = 54,
         GetEnvironmentVariable = 55,
-        MonitorApiKeyNotConfigured = 56,
-        QueueDoesNotExist = 57,
-        QueueOptionsPartiallySet = 58,
-        WritingMessageToQueueFailed = 59,
+        MonitorApiKeyNotConfigured = 56, // Note the gap - from removing things related to Azure egress
```

Comment:
Will we want to update all of these numbers to be in-order again? I haven't for now, but we have several gaps left behind by the logging specific to the Azure extension.

Reply:
Do not remove or reorder anything here. Just add a comment after then that they've moved to the extension.